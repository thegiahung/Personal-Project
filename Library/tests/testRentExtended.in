rent 100000 111111
add member Python
add member Java
add member Ruby
add collection books.csv
rent 100000 111111
relinquish 100000 111111
rent 100001 111111
rent 100002 111111
rent 100002 111112
rent 100002 111112
rent 100002 120000
exit
