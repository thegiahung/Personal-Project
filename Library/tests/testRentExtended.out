user: No members in system.

user: Success.

user: Success.

user: Success.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Book is currently unavailable.

user: No such book in system.

user: Ending Library process.
